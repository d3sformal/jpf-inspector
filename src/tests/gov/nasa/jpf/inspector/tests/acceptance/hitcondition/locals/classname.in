cr bp class = *Callstack
run && wait
run && wait
del bp 1
run && wait