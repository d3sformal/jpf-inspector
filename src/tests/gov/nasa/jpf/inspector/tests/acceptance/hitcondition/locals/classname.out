INFO: JPF created and connected, SuT is started
INFO: Breakpoint hit: [1 :  state=ENABLED hits=1 hitsTotal=1 class = *Callstack]
	In class "gov.nasa.jpf.inspector.tests.acceptance.sut.Callstack".
INFO: Breakpoint hit: [1 :  state=ENABLED hits=2 hitsTotal=2 class = *Callstack]
	In class "gov.nasa.jpf.inspector.tests.acceptance.sut.Callstack".
Breakpoint with ID 1 successfully deleted.
INFO: JPF Terminating